<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>508</width>
    <height>398</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(201, 255, 188);</string>
  </property>
  <widget class="QPushButton" name="btn2_0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>250</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1_0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>170</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>170</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0_1">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0_0">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>81</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Showcard Gothic</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(255, 255, 255);

  border-style: dotted;
 border-radius: 15px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Marker</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Marker</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_player1">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Marker</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_player2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Marker</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_pp">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>350</y>
     <width>151</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Marker</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player vs Player</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rb_pc">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>350</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe Marker</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Player vs CPU</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Orange LET</family>
     <pointsize>26</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(85, 85, 255);</string>
   </property>
   <property name="text">
    <string>Tic-Tac-Toe</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
